docs/_website/docusaurus.config.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
     'https://fonts.googleapis.com/css2?family=Roboto&family=Open+Sans&display=swap',
     'https://cdn.rawgit.com/luizbills/feather-icon-font/v4.7.0/dist/feather.css',
   ],
-  plugins: ['@docusaurus/plugin-google-analytics'],
+  plugins: [],
```

Comment:
confirmed that GA is still included in page. the build was throwing an error about duplicate plugins before this change. guessing it's auto-included from the core theme.

Reply:
huh that's awesome. They include the package as a dep in core so this makes sense.